routes/accounts.js

```diff
@@ -80,10 +80,22 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var sessionId = toBearerToken(request)
 
-      return accounts.findAll({
-        db: options.db,
-        bearerToken: sessionId,
-        include: request.query.include
+      admins.validateSession(sessionId)
+
+      .catch(function (error) {
+        if (error.name === 'not_found') {
```

Comment:
@gr2m I'm not a big fan of this line... comparing a field with a string constant. Would be better to require the module in pouchDB */src/deps/errors.js* and compare against the real error object? Maybe it's my background (not enough experience with node), but I like that better than comparing with string constants. However, it appears that is more common practice in this project and I don't want to do something radical here...